api/src/paths/project/{projectId}/survey/{surveyId}/attachments/upload.ts

```diff
@@ -74,7 +64,7 @@ POST.apiDoc = {
             },
             attachmentType: {
               type: 'string',
-              enum: ['Report', 'Other']
+              enum: ['Other']
             },
```

Comment:
We can actually remove this entirely from both the upload endpoints, now that we split them out